templates/download/iot/raspberry-pi-2-3.html

```diff
@@ -40,28 +43,27 @@ <h3>Minimum requirements</h3>
     <div class="col-12">
       <h2>Installation instructions</h2>
       <ol class="p-stepped-list--detailed">
-        {% include "./_setup-ubuntu-sso.html" with number=1 %}
         <li class="p-list-step__item">
           <h3 class="p-list-step__title">
-            <span class="p-list-step__bullet">2</span>
-            Download Ubuntu Core
+            <span class="p-list-step__bullet">1</span>
+            Download Ubuntu Server
           </h3>
           <div class="p-list-step__content">
-            <p>Get the correct Ubuntu Core image for your board:</p>
+            <p>Get the correct Ubuntu Server image for your board:</p>
             <ul class="u-no-margin--left">
-              <li><a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/ubuntu-core-18-armhf+raspi2.img.xz">Ubuntu Core 18 image for Raspberry Pi 2.</a>
+              <li><a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/ubuntu-{{ lts_release_with_point }}-preinstalled-server-armhf+raspi2.img.xz">Ubuntu Server image for Raspberry Pi 2.</a>
               </li>
-              <li><a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/ubuntu-core-18-armhf+raspi3.img.xz">Ubuntu Core 18 image for Raspberry Pi 3.</a></li>
-              <li>You can verify the integrity of the files using the <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/SHA256SUMS">SHA256SUM</a> and <a class="p-link--external" href="http://cdimage.ubuntu.com/ubuntu-core/18/stable/current/SHA256SUMS.gpg">SHA256SUM.gpg</a> files.
+              <li><a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/ubuntu-{{ lts_release_with_point }}-preinstalled-server-arm64+raspi3.img.xz">Ubuntu Server image for Raspberry Pi 3</a></li>
+              <li>You can verify the integrity of the files using the <a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/SHA256SUMS">SHA256SUM</a> and <a class="p-link--external" href="http://cdimage.ubuntu.com/releases/bionic/release/SHA256SUMS.gpg">SHA256SUM.gpg</a> files.
               </li>
             </ul>
           </div>
         </li>
-        {% include "./_flash-image.html" with card="microSD card" number=3 %}
+        {% include "./_flash-image.html" with card="microSD card" number=2 %}
```

Comment:
Here we need to update the include text from "Copy the Ubuntu Core image on the microSD" to "Copy the Ubuntu image on the microSD"

Reply:
I removed core from every mention